tests/test_credibility_check.py

```diff
@@ -0,0 +1,58 @@
+from typing import Dict
+
+from lunavl.sdk.estimators.face_estimators.facewarper import FaceWarpedImage
+from lunavl.sdk.estimators.face_estimators.credibility_check import CredibilityCheck, CredibilityCheckEstimator
+from lunavl.sdk.image_utils.image import VLImage
+
+from tests.base import BaseTestClass
+from tests.resources import WARP_CLEAN_FACE
+from tests.schemas import jsonValidator, CREDIBILITY_CHECK_SCHEMA
+
+
+class TestCredibilityCheck(BaseTestClass):
+    """
+    Test estimate credibility check.
+    """
+
+    credibilityCheckEstimator: CredibilityCheckEstimator
+
+    @classmethod
+    def setup_class(cls):
+        super().setup_class()
+        cls.credibilityCheckEstimator = cls.faceEngine.createCredibilityCheckEstimator()
+
+        cls.warp = FaceWarpedImage(VLImage.load(filename=WARP_CLEAN_FACE))
+
+    def assertrCedibilityCheckEstimation(
+        self, credibilityCheck: CredibilityCheck, expectedEstimationResults: Dict[str, str]
+    ):
+        """
+        Function checks if the instance belongs to the credibility check class 
+        and compares the result with what is expected.
+
+        Args:
+            credibilityCheck: credibility check estimation object
+            expectedEstimationResults: dictionary with result
+        """
+        assert isinstance(credibilityCheck, CredibilityCheck), f"{credibilityCheck.__class__} is not {CredibilityCheck}"
```

Comment:
add check for result type (`isinstance() -> float`) and value in range [0,1]

Reply:
Doesn't this check 'CREDIBILITY_CHECK_SCHEMA' schema? There, in the properties is used TYPE_SCORE, which checks the type and limit of the value

Reply:
the numeric type for json schema is not equal a python type object

Reply:
fixed